app/Services/ModuleService.php

```diff
@@ -70,4 +81,139 @@ public function getAdminLinks(): array
     {
         return self::$adminLinks;
     }
+
+    public function createModule($array): bool
+    {
+        $orig_file = $array['file'];
```

Comment:
Add documentation to the method here

```
/**
  * @param array $array Valid fields are x, y,, z
 */

Reply:
`Valid fields are x, y,, z`
I wonder what does that mean? The fields in array?

Reply:
Just ignore this one... I think just rename this and change the parameters like I described in the other comment.

Reply:
Alright!